upi/azurestack/04_bootstrap.json

```diff
@@ -0,0 +1,266 @@
+{
+  "$schema" : "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion" : "1.0.0.0",
+  "parameters" : {
+    "baseName" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Base name to be used in resource names (usually the cluster's Infra ID)"
+      }
+    },
+    "bootstrapIgnition" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Bootstrap ignition content for the bootstrap cluster"
+      }
+    },
+    "sshKeyData" : {
+      "type" : "securestring",
+      "metadata" : {
+        "description" : "SSH RSA public key file as a string."
+      }
+    },
+    "diagnosticsStorageAccountName": {
+      "type": "string"
+    },
+    "bootstrapVMSize" : {
+      "type" : "string",
+      "defaultValue" : "Standard_D4_v2",
+      "allowedValues" : [
```

Comment:
We are finding in AWS managing this list is a maintenance burden. Do we want to inherit the same burden here? I understand the tradeoff is increasing customer success (as long as are maintaining the list).

Reply:
I believe this list will need to be maintained by docs, so this would be n+1 places where we will have to keep things in sync.

Reply:
I'll just delete the list, and we can add it later if we need to.